src/main/java/com/example/task_2/bus_ticket/BusTicketService.java

```diff
@@ -0,0 +1,41 @@
+package com.example.task_2.bus_ticket;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BusTicketService {
+
+    private final List<BusTicket> storage = new ArrayList<>();
+
+    public void addTicket(BusTicket ticket) {
+        storage.add(ticket);
+    }
+
+    public BusTicket getTicketById(String ticketId) {
+        for (BusTicket ticket : storage)
+            if (ticket.getId().toString().equals(ticketId))
+                return ticket;
```

Comment:
it is better to always include {} quotes, even if statement body is one line like here